### PR TITLE
Create ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by pry

Ticket: [MTM-57553](https://cumulocity.atlassian.net/browse/MTM-57553)
Original PR: [6201](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6201)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed breadcrumbs for global and inventory roles tabs under Accounts > Roles [GRAFT][release/cd] (#6201)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed breadcrumbs for global and inventory roles tabs under Accounts > Roles
```

File: content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57553
+version: 1019.22.3
+---
+Fixed breadcrumbs for global and inventory roles tabs under Accounts > Roles [GRAFT][release/cd] (#6201)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT user interface, the breadcrumbs for the global and inventory roles tabs under the Accounts > Roles menu were previously incorrect, making navigation confusing for users. This issue has been resolved, and the breadcrumbs now accurately reflect the user's location within the application. This change improves the overall user experience by providing clear and consistent navigation across the roles management screens.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by pink-mongoose — CHANGES_REQUESTED** [review]
This needs changing so it makes sense for a user.

In the Cumulocity IoT Administration App, the location information for the global and inventory roles tabs under the Accounts > Roles menu was incorrect, making navigation confusing for users. This issue has been resolved with the user's location within the application now accurately reflected.

File: content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57553
+version: 1019.22.3
+---
+In the Administration application, the location information for the **Global roles**  and **Inventory roles** tabs under the **Accounts** > **Roles** menu was incorrect, making navigation confusing for users. This issue has been resolved with the user's location within the application now accurately reflected.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md:17]:
```suggestion
In the Administration application, the location information for the **Global roles** and **Inventory roles** tabs under the **Accounts** > **Roles** menu was incorrect, making navigation confusing for users. This issue has been resolved with the user's location within the application now accurately reflected.
```

**Comment by BeateRixen** [comment]
@martin-sag @Airdroplet Please review and approve

**Comment by martin-sag** [comment]
Is there a ways to find out who originally created the PR? Not sure if the corresponding developer gets all those notifications and change requests.

**Review by Airdroplet — APPROVED** [review]

**Review by martin-sag — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed location information for Global roles and Inventory roles tabs under Accounts > Roles menu
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md:3]:
```suggestion
title: Fixed breadcrumbs for Global roles and Inventory roles tabs
```

File: content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57553
+version: 1019.22.3
+---
+In the Administration application, the location information for the **Global roles** and **Inventory roles** tabs under the **Accounts** > **Roles** menu was incorrect, making navigation confusing for users. This issue has been resolved with the user's location within the application now accurately reflected.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md:17]:
```suggestion
In the Administration application, the location information for the **Global roles** and **Inventory roles** tabs under **Accounts** > **Roles** was incorrect, making navigation confusing for users. This issue has been resolved with the user's location within the application now accurately reflected.
```

File: content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed breadcrumbs for Global roles and Inventory roles tabs
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-22-3-fixed-breadcrumbs-for-global-and-inventory-roles-tabs-under-accounts-roles.md:3]:
```suggestion
title: Fixed location information for Global roles and Inventory roles tabs
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
> Is there a ways to find out who originally created the PR? Not sure if the corresponding developer gets all those notifications and change requests.

@martin-sag Since these PRs are automatically created they do not include the developer. @janhommes would it be possible to automatically add the developer who merged the change?

**Pull request merged**